ERCS/erc-7694.md

```diff
@@ -0,0 +1,110 @@
+---
+eip: 7694
+title: Solana storage handler for CCIP-Write
+description: Cross-chain write deferral protocol incorporating storage handler for Solana
+author: Avneet Singh (@sshmatrix), 0xc0de4c0ffee (@0xc0de4c0ffee)
+discussions-to: https://ethereum-magicians.org/t/eip-7694-solana-storage-handler-for-ccip-write/19706
```

Comment:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7694-solana-storage-handler-for-ccip-write/19706
```

Reply:
Fixed in https://github.com/ethereum/ERCs/pull/390/commits/7a09d42beab435c3fc751734c99ec41e367184d7